scipy/stats/mstats_basic.py

```diff
@@ -2515,31 +2515,43 @@ def _stdemed_1D(data):
 SkewtestResult = namedtuple('SkewtestResult', ('statistic', 'pvalue'))
 
 
-def skewtest(a, axis=0):
+def skewtest(a, axis=0, alternative='two-sided'):
     """
     Tests whether the skew is different from the normal distribution.
 
     Parameters
     ----------
-    a : array
+    a : array_like
         The data to be tested
     axis : int or None, optional
        Axis along which statistics are calculated. Default is 0.
        If None, compute over the whole array `a`.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+          * 'two-sided'
+          * 'less': one-sided
+          * 'greater': one-sided
+
+        .. versionadded:: 1.7.0
 
     Returns
     -------
-    statistic : float
+    statistic : array_like
         The computed z-score for this test.
-    pvalue : float
-        a 2-sided p-value for the hypothesis test
+    pvalue : array_like
+        A p-value for the hypothesis test
 
     Notes
     -----
     For more details about `skewtest`, see `stats.skewtest`.
 
     """
     a, axis = _chk_asarray(a, axis)
+    if alternative not in {'two-sided', 'less', 'greater'}:
+        raise ValueError("alternative must be "
```

Comment:
Looks like codecov is saying the error handling code isn't tested here.

Reply:
Oops, forgot to test the error! Now, done. Thanks.